src/App.js

```diff
@@ -1,11 +1,37 @@
-
 import './App.css';
 
+import { Routes, Route } from "react-router-dom"
+
+import Header from './components/Header';
+import Navbar from './components/Navbar';
+import Home from './components/Home';
+import Articles from './components/Articles';
+
+import MyAccount from './components/MyAccount';
+import Footer from './components/Footer';
+import LoginBar from './components/UserLoginStatusBar';
+
+
+
 function App() {
+
+
+  
   return (
+
     <div className="App">
+        <Header />
+        <Navbar />
+        <LoginBar />
+        <Routes>
+                <Route path="/" element={<Home />}></Route>
+                <Route path="/articles/*" element={<Articles />}></Route>
```

Comment:
You'll need to change this endpoint once you get to working on displaying a single article. We want it to show all articles upon receiving /articles, and show a whole, single article when we go to /articles/:article_id - at the moment, if the user went to an endpoint for a single article (eg: /articles/1), or even /articles/banana - they would still get the list of all articles.